.eslintrc

```diff
@@ -1,7 +1,5 @@
-extends: universal-search
+extends: universal-search/addon
 
 rules:
-  consistent-return: 0
   no-console: 0
-  no-var: 2
-  prefer-const: 2
+  func-names: 0
```

Comment:
[`func-names`](http://eslint.org/docs/rules/func-names.html) is a funny rule (via [**eslint-config-airbnb**](https://github.com/airbnb/javascript/blob/60480c05ea6c2d0f172302f845c39f5f62accb8d/packages/eslint-config-airbnb/rules/style.js#L19-L20)) which throws ~60 warnings in this repo if you have anonymous functions.

Oddly, I only see the warnings if I explicitly run ESLint and they don't seem to appear when I run `npm run lint` (which uses Gulp).